### PR TITLE
Add "Send password reset email" button to backend users update page

### PR DESCRIPTION
**Pull request by prsuhas**
Fix for issue #654 

Added "Send password reset email" button to backend users update page.

@LukeTowers I have re-used the existing Password reset logic present in `modules/backend/controllers/Auth.php` file. Should I create a helper function to remove code duplication ?

### CODE REVIEW

File: modules/backend/controllers/users/_btn_password_reset.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+        type="button"
+        data-request="onManualPasswordReset"
+        data-load-indicator="<?= e(trans('backend::lang.account.working')) ?>"
+        data-request-confirm="<?= e(trans('backend::lang.account.manual_password_reset_confirm')) ?>"
+        class="btn btn-warning wn-icon-key"
+        style="width: 100%; text-align: center"
+    >
+        <?= e(trans('backend::lang.account.password_reset_email')) ?>
+    </button>
+</div>
```
Comment (LukeTowers) [inline modules/backend/controllers/users/_btn_password_reset.php]:
```suggestion
</div>

```

File: modules/backend/controllers/users/_btn_password_reset.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<div class="loading-indicator-container">
+    <button
+        type="button"
+        data-request="onManualPasswordReset"
+        data-load-indicator="<?= e(trans('backend::lang.account.working')) ?>"
```
Comment (LukeTowers) [inline modules/backend/controllers/users/_btn_password_reset.php]:
```suggestion
        data-load-indicator="<?= e(trans('backend::lang.account.sending')) ?>"
```

File: modules/backend/lang/en/lang.php [hunk 3]
```diff
@@ -75,6 +75,10 @@
         'cancel' => 'Cancel',
         'delete' => 'Delete',
         'ok' => 'OK',
+        'working' => 'Working...',
```
Comment (LukeTowers) [inline modules/backend/lang/en/lang.php]:
```suggestion
        'sending' => 'Sending...',
```

File: modules/backend/lang/en/lang.php [hunk 4]
```diff
@@ -75,6 +75,10 @@
...
+        'password_reset_email' => 'Send password reset email',
+        'manual_password_reset_confirm' => 'Are you sure you want to send password reset mail for this user?',
```
Comment (LukeTowers) [inline modules/backend/lang/en/lang.php]:
```suggestion
        'manual_password_reset_confirm' => 'Are you sure you want to send a password reset email to this user?',
```

File: modules/backend/lang/en/lang.php [hunk 5]
```diff
@@ -75,6 +75,10 @@
...
+        'manual_password_reset_success' => 'An email has been sent to user with reset instructions.',
```
Comment (LukeTowers) [inline modules/backend/lang/en/lang.php]:
```suggestion
        'manual_password_reset_success' => 'An email has been sent to the user with instructions to reset their password.',
```

File: modules/backend/controllers/Users.php [hunk 6]
```diff
@@ -233,4 +234,32 @@ protected function generatePermissionsField()
...
     }
+
+    /**
+     * Send password reset mail
+     */
+    public function update_onManualPasswordReset($recordId)
+    {
+        $user = $this->formFindModelObject($recordId);
+
+        //@todo::Force Trusted Host verification
```
Comment (LukeTowers) [inline modules/backend/controllers/Users.php]:
```suggestion
```

**Review by LukeTowers — CHANGES_REQUESTED** [review]

**Review by LukeTowers — APPROVED** [review]

**Comment by LukeTowers** [comment]
@prsuhas no, that's fine, we can leave the code copy-pasted for now. Can you add a screenshot of what the new button looks like please?

**Comment by prsuhas** [comment]
> @prsuhas no, that's fine, we can leave the code copy-pasted for now. Can you add a screenshot of what the new button looks like please?

Please find below screenshot

Screenshot from 2022-10-03 09-53-54

Pushed the review changes too.

**Pull request merged**